label_studio/tasks/models.py

```diff
@@ -65,6 +65,8 @@ class Task(TaskMixin, models.Model):
         'data_import.FileUpload', on_delete=models.SET_NULL, null=True, blank=True, related_name='tasks',
         help_text='Uploaded file used as data source for this task'
     )
+    inner_id = models.BigIntegerField(_('inner id'), default=0, db_index=True,
```

Comment:
It's safer to add only null=True columns to the existing database, because migration, that tries to fill the big table with default values can lock the database and be the reason for production downtime.

If we need to have null=False constraint it can be added by the second migration.